### DISCUSSION TITLE
glue 의 schema-registry는 어떤 장점이 있을까?

### DISCUSSION THREAD

**Discussion by kyungjunleeme** [kyungjunleeme/kafka_study#41] [Q&A]
https://docs.aws.amazon.com/ko_kr/glue/latest/dg/schema-registry.html

**Comment by hellices** [comment]
제가 발표했던 내용에서 잠깐 소개드렸는데, cdc connector를 schema-registry없이 사용하는 경우 meta정보를 json-schema(defaut)로 토픽 안에 offset별로 계속해서 전송합니다. schema-registry를 사용하는 경우 meta정보를 registry에서 가져와서 비교하기 때문에 토픽 사이즈가 1/10로 줄어듭니다.
kafka-client 등으로 연동하더라도 전송 시에 schema-registry와 통합하게 되면 producer레벨에서 스펙을 비교할 수 있게 되어 버그 발생 확률을 줄어들 거라고 생각합니다.
